src/benchmarks/real-world/Roslyn/CompilerBenchmarks.csproj

```diff
@@ -6,7 +6,11 @@
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp3.0</TargetFramework>
     <IsShipping>false</IsShipping>
+    <!-- this app is using internal Roslyn API and needs to be signed -->
+    <SignAssembly>true</SignAssembly>
```

Comment:
Will this cause test signing or real signing?

Reply:
I don't know, but I can explain why this change is needed now. So far Roslyn benchmarks were not referencing the `common.props` file and they were using the `Directory.Build.props` file from the repository root folder in an implicit way. This particular `Directory.Build.props` was importing Arcade SDK which turns on signing by default.

We don't want to sign MicroBenchmarks.csproj (we don't need it, it also has dependencies to 3rd party not signed libs like Jil), but since we have renamed `common.props` to `Directory.Build.props` it's now going to get imported by Roslyn benchmarks.

https://github.com/dotnet/performance/blob/22fe7870256de2a3b72b94ca2259be2bcdf72587/src/common.props#L27-L28 

And we need to enable it here in explicit way.